AirCasting/Models/AirBeamFixedSessionCreator.swift

```diff
@@ -47,21 +47,23 @@ final class AirBeamFixedSessionCreator: SessionCreator {
               let startTime = session.startTime,
               let peripheral = sessionContext.peripheral,
               let wifiSSID = sessionContext.wifiSSID,
-              let wifiPassword = sessionContext.wifiPassword else {
+              let wifiPassword = sessionContext.wifiPassword,
+              let contribute = sessionContext.contribute,
+              let isIndoor = sessionContext.isIndoor else {
```

Comment:
Is it based on what user selects when creating fixed sessions? It should be :)

Reply:
It is 😄